https://bugs.gentoo.org/421649

For some reason testsuite does not handle warnings well:
  /tmp/lt-cZI2cFo0z7.c:2:24: warning: inplicit function declaration «puts» [-Wimplicit-function-declaration]
       void func2(void) { puts("func2"); }
                          ^
and abandons problematic tests:
  Testcase compile failed, so all tests in this file will automatically fail.
diff --git a/testsuite/ltrace.main/filters.exp b/testsuite/ltrace.main/filters.exp
index 988346f..f7f4140 100644
--- a/testsuite/ltrace.main/filters.exp
+++ b/testsuite/ltrace.main/filters.exp
@@ -24,2 +24,3 @@ set libfilt1 [ltraceCompile libfilt1.so [ltraceSource c {
 set libfilt2 [ltraceCompile libfilt2.so [ltraceSource c {
+    #include <stdio.h>
     void func2(void) { puts("func2"); }
diff --git a/testsuite/ltrace.main/parameters.c b/testsuite/ltrace.main/parameters.c
index ff24a38..9569dbe 100644
--- a/testsuite/ltrace.main/parameters.c
+++ b/testsuite/ltrace.main/parameters.c
@@ -19,2 +19,3 @@ void func_strfixed(char*);
 void func_ppp(int***);
+void func_string(char*);
 void func_stringp(char**);
diff --git a/testsuite/ltrace.main/signals.c b/testsuite/ltrace.main/signals.c
index a02e795..fda4ab9 100644
--- a/testsuite/ltrace.main/signals.c
+++ b/testsuite/ltrace.main/signals.c
@@ -7,2 +7,3 @@
 #include <sys/types.h>
+#include <unistd.h>
 
diff --git a/testsuite/ltrace.minor/time-record.c b/testsuite/ltrace.minor/time-record.c
index a66b838..7d5e5e3 100644
--- a/testsuite/ltrace.minor/time-record.c
+++ b/testsuite/ltrace.minor/time-record.c
@@ -7,2 +7,3 @@
 #include <time.h>
+#include <unistd.h>
 
diff --git a/testsuite/ltrace.minor/trace-clone.c b/testsuite/ltrace.minor/trace-clone.c
index db1936d..c68b128 100644
--- a/testsuite/ltrace.minor/trace-clone.c
+++ b/testsuite/ltrace.minor/trace-clone.c
@@ -10,2 +10,3 @@
 #include <sched.h>
+#include <unistd.h>
 
diff --git a/testsuite/ltrace.minor/trace-fork.c b/testsuite/ltrace.minor/trace-fork.c
index c5f0c71..e1ab17f 100644
--- a/testsuite/ltrace.minor/trace-fork.c
+++ b/testsuite/ltrace.minor/trace-fork.c
@@ -8,2 +8,4 @@
 #include <sys/types.h>
+#include <sys/wait.h>
+#include <unistd.h>
 
@@ -29,3 +31,4 @@ main ()
       printf("My child pid is %d\n",pid);
-      wait(); 
+      int status;
+      wait(&status);
     }
diff --git a/testsuite/ltrace.torture/signals.c b/testsuite/ltrace.torture/signals.c
index b786c81..86e2dba 100644
--- a/testsuite/ltrace.torture/signals.c
+++ b/testsuite/ltrace.torture/signals.c
@@ -7,2 +7,3 @@
 #include <sys/types.h>
+#include <unistd.h>
 
